network/wsNetwork.go

```diff
@@ -1463,8 +1464,11 @@ func (wn *WebsocketNetwork) preparePeerData(request broadcastRequest, prio bool,
 			digests[i] = crypto.Hash(mbytes)
 		}
 
-		if prio && request.tags[i] == protocol.ProposalPayloadTag {
-			networkPrioPPNonCompressedSize.AddUint64(uint64(len(d)), nil)
+		if prio {
+			if request.tags[i] == protocol.ProposalPayloadTag {
```

Comment:
Is it necessary to make to levels of if?

Reply:
yes, we want containsPrioPPTag